css-properties-values-api/Overview.bs

```diff
@@ -933,41 +934,43 @@ The syntax of ''@property'' is:
 	}
 	</pre>
 
-''@property'' rules require a 'syntax', and 'inherits' descriptor; if either are
-missing, the ''@property'' rule is invalid, and must not result in a registration.
-The 'initial-value' descriptor is optional only if the syntax is the
-[=universal syntax descriptor=], otherwise the descriptor is required; if it's
-missing, the ''@property'' rule is invalid,
-and must not result in a registration.
+''@property'' rules require a 'syntax' and 'inherits' descriptor;
+if either are missing,
+the entire rule is invalid and must be ignored.
+The 'initial-value' descriptor is optional
+only if the syntax is the [=universal syntax descriptor=],
+otherwise the descriptor is required;
+if it's missing, the entire rule is invalid and must be ignored.
 
-The ''@property'' rule accepts the descriptors defined in this chapter.
-Declarations that contain other descriptors not defined here, must be ignored to
-maintain forwards compatiblity.
+If an ''@property'' rule contains any unknown descriptors,
+the entire rule is invalid and must be ignored.
```

Comment:
@tabatkins Ouch, my forwards compat! :scream:

Reply:
Oh, I misread what you'd wrote there!